src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -13226,7 +13226,7 @@ avtSiloFileFormat::GetPointMesh(DBfile *dbfile, const char *mn, int domain)
         // so that it can be obtained through the GetAuxiliaryData call
         //
         void_ref_ptr vr = void_ref_ptr(arr, avtVariableCache::DestructVTKObject);
-        cache->CacheVoidRef(meshname, AUXILIARY_DATA_GLOBAL_NODE_IDS, timestep, 
+        cache->CacheVoidRef(meshname, AUXILIARY_DATA_GLOBAL_ZONE_IDS, timestep, 
                             domain, vr);
```

Comment:
This doesn't impact existing global node id logic?

Reply:
@markcmiller86 this change is confusing me, too.

Reply:
Ok, thanks for catching this. Reverted in 614f051b84c5fcbb6098383a3ea8f8b9a9fb45ed